test/avocado/seleniumlib.py

```diff
@@ -169,13 +172,13 @@ def wait(self, method, text, baseelement, overridetry, fatal, cond, jscheck):
             if fatal:
                 # sample screenshot name is: screenshot-test20Login.png
                 # it stores super caller method to name via inspection code stack
-                screenshot_file = "screenshot-%s.png" % str(inspect.stack()[2][3])
+                screenshot_file = "screenshot%s.png" % str(inspect.stack()[2][3])
                 additional_text = ""
                 try:
-                    self.driver.save_screenshot(screenshot_file)
+                    self.driver.save_screenshot(os.path.join(str(actualpath),screenshot_file))
                     self.error = False
-                except:
-                    screenshot_file = "Unable to catch screenshot: " + screenshot_file
+                except Exception as e:
+                    screenshot_file = "Unable to catch screenshot: " + screenshot_file + str(e)
```

Comment:
you can avoid str() wrapper by using something like
`"Unable to catch screenshot: {0} ({1})".format(screenshot_file, e)`